packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheetStep.js

```diff
@@ -70,7 +70,6 @@ export let CreateTearsheetStep = forwardRef(
         stepNumber === stepsContext?.currentStep &&
         previousState?.currentStep !== stepsContext?.currentStep
       ) {
-        stepsContext?.setOnNext(onNext);
         stepsContext?.setOnMount(onMount);
       }
     }, [onMount, onNext, stepsContext, stepNumber, previousState?.currentStep]);
```

Comment:
The above commentary also needs updating

```suggestion
    }, [onMount, stepsContext, stepNumber, previousState?.currentStep]);
```

Reply:
Updated